### DISCUSSION TITLE
關於Upload組件疑問請教

### DISCUSSION THREAD

**Discussion by Ya-wenWu** [dotnetcore/BootstrapBlazor#1492] [Q&A] [answered]
您好，
請問關於˙Upload組件, 在開啟檔案總管時,是否有幫忙過濾副檔名的attribute?
除了inputfileupload 組件外, 另外兩個組件是否能過濾副檔名呢？
謝謝您

**Comment by ArgoZhang** [comment]
@Ya-wenWu 请问副档名是什么意思

**Reply by Ya-wenWu** [reply]
@ArgoZhang  您好! 是指檔案名稱中，「.」後面的名稱。舉例如「example.txt」的這個檔名中，example是主檔名，txt為副檔名。

**Reply by densen2014** [reply, marked as answer]
参考

```

<input  accept="image/*" multiple="" type="file" > 

```  

其中 accept="image/*" 就是你要的答案

**Reply by Ya-wenWu** [reply]
@densen2014 非常感謝您!
